osu.Game/Localisation/AudioSettingsStrings.cs

```diff
@@ -32,6 +32,11 @@ public static class AudioSettingsStrings
         /// <summary>
         /// "Master"
         /// </summary>
+        public static LocalisableString PositionalLevel => new TranslatableString(getKey(@"positional_hitsound_audio_level"), @"Positional hitsound audio level.");
```

Comment:
not sure how this happened but the xmldocs here are completely wrong. did you add this entry manually? it should not be done manually, there is tooling that is supposed to do it automatically for you.

additionally, no other string in this file is terminated with a full stop:

```suggestion
        public static LocalisableString PositionalLevel => new TranslatableString(getKey(@"positional_hitsound_audio_level"), @"Positional hitsound audio level");
```